blazor/rich-text-editor/getting-started.md

```diff
@@ -199,164 +199,7 @@ Configure the toolbar with the tools using [RichTextEditorToolbarSettings.Items]
 {% endhighlight %}
 {% endtabs %}
 
-{% previewsample "https://blazorplayground.syncfusion.com/embed/hthTNWAZUAqIpYkv?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" backgroundimage "[Blazor RichTextEditor with Toolbar](./images/blazor-richtexteditor-toolbar.png)" %}
-
-## Insert images and links
-
```

Comment:
Dont remove it in blazor @Harish-SF4430 


We will remove after full docs correction